message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -183,9 +183,9 @@
       </entry>
       <entry value="42502" name="MAV_CMD_GIMBAL_AXIS_CALIBRATION_STATUS">
         <description>Reports progress and success or failure of gimbal axis calibration procedure.</description>
-        <param index="1">Gimbal axis we're reporting calibration progress for.</param>
-        <param index="2">Current calibration progress for this axis, 0x64=100%.</param>
-        <param index="3">Status of the calibration.</param>
+        <param index="1" label="axis" enum="GIMBAL_AXIS">Gimbal axis we're reporting calibration progress for.</param>
+        <param index="2" label="progress" units="%">Current calibration progress for this axis.</param>
```

Comment:
I assume we can ignore attributes where they can be assumed? Ie a percentage by default might be assumed to have minvalue of 0 and max of 100

Reply:
improved